public/app.d.ts

```diff
@@ -1505,6 +1505,20 @@ declare namespace JMap {
          * ```
          */
         function getHeight(): number
+
+        /**
+         * **JMap.Application.UI.Container.getContainerCoordinates**
+         * 
+         * Returns the coordinates (five points) in pixel of the closing polygon representing the dom container. .
```

Comment:
Returns a polygon geometry representing the visible coordinates on the map.
Works fine even if the map is pitched or rotated.